web-components/src/[sandbox]/examples/draggable.ts

```diff
@@ -0,0 +1,288 @@
+              <md-icon class="handle" name="panel-control-dragger_16"></md-icon>
+              <div class="first">Agent Name 333</div>
+              <div class="second">
+                <md-checkbox checked>Left Option</md-checkbox>
+              </div>
+              <div class="third">
+                <md-combobox .options=${comboBoxOptions} placeholder="Choose options" ></md-combobox>
+              </div>
+              <div class="fourth">
+                <md-icon name="plus-circle_24" size="16"></md-icon>
+                <span>Add Field Filter</span>
+              </div>
+            </div>
+          </md-list-item>
+          <md-list-item slot="list-item">
+            <div class="md-row">
+              <md-icon class="handle" name="panel-control-dragger_16"></md-icon>
+              <div class="first">Agent Name 5555</div>
+              <div class="second">
+                <md-checkbox checked>Left Option</md-checkbox>
+              </div>
+              <div class="third">
+                <md-combobox .options=${comboBoxOptions} placeholder="Choose options" ></md-combobox>
+              </div>
+              <div class="fourth">
+                <md-icon name="plus-circle_24" size="16"></md-icon>
+                <span>Add Field Filter</span>
+              </div>
+            </div>
+          </md-list-item>
+        </md-list>
+      </md-draggable>
+    `;
+  }
+}
+
+@customElement("shared-draggable-sandbox")
+export class SharedDraggable extends LitElement {
+  static get styles() {
+    return [
+      reset,
+      css`
+      .shared-draggable-wrapper {
+        display: flex;
+        justify-content: space-around;
+      }
+      .sorting {
+        opacity: 0.4;
+        border-style: dashed;
+        border-color: #333;
+        background-color: #fff;
+      }
+      md-list-item[selected]::part(list-item),
+      md-list-item[draggable="true"]::part(list-item) {
+        background: #fff !important;
+      }
+    }`
+    ];
+  }
+
+  render() {
+    return html`
+      <div class="shared-draggable-wrapper">
+        <md-draggable
+          draggable-items="md-list-item"
+          .group=${{ name: "md-list", pull: "clone" } as GroupOptions}
+          filter="md-list-item[disabled]"
+        >
+          <md-list label="Transuranium elements">
+            <md-list-item slot="list-item">
+              <md-icon name="tag_16"></md-icon>
+              Average CSAT Scores
+            </md-list-item>
+            <md-list-item slot="list-item">
+              <md-icon name="tag_16"></md-icon>
+              Average Handle Time
+            </md-list-item>
+            <md-list-item slot="list-item">
+              <md-icon name="tag_16"></md-icon>
+              Total Contacts Handled
+            </md-list-item>
+            <md-list-item slot="list-item">
+              <md-icon name="tag_16"></md-icon>
+              Internal Filter
+            </md-list-item>
+            <md-list-item slot="list-item">
```

Comment:
these guys need to be droppable into another draggable container and turn into a draggable row

Reply:
Updated